client/environment.go

```diff
@@ -122,6 +122,7 @@ type Environment struct {
 	LatestDeploymentLog         DeploymentLog `json:"latestDeploymentLog"`
 	TerragruntWorkingDirectory  string        `json:"terragruntWorkingDirectory,omitempty"`
 	VcsCommandsAlias            string        `json:"vcsCommandsAlias"`
+	VcsPrCommentsEnabled        bool          `json:"vcsPrCommentsEnabled" tfschema:"-"`
```

Comment:
@TomerHeber can you explain more?
we indeed need that- 
if a customer
1. use the the provider version
2. do not set `vcsPrCommentsEnabled`
3. do set `VcsCommandsAlias`

then the BE response may return `vcsPrCommentsEnabled` `true` in the response.
we should not make a drift for that.

use case - a customer that used the alias in old provider version, then did an upgrade.

Reply:
yeah - in the schema it will be false.
But in the API it will pass true... because alias is used in the schema.
There's a part where this drift is ignored.

Reply:
The answer has changed.
the return value will now be ignored if nothing was configured in the schema. Not a perfect solution, but should work well for customers.